language/move-core/types/src/vm_status.rs

```diff
@@ -635,6 +635,7 @@ pub enum StatusCode {
     CALL_STACK_OVERFLOW = 4021,
     VM_MAX_TYPE_DEPTH_REACHED = 4024,
     VM_MAX_VALUE_DEPTH_REACHED = 4025,
+    STDLIB_UPGRADE_ERROR = 4026,
```

Comment:
One last change please. The convention for 0L changes is to add a comment above or next to the line, so we know what we have changed for purposes of pulling upstream changes.

```
//////// 0L ////////
```